### ISSUE TITLE
Missing Bio.Alphabet Dependency

### ISSUE THREAD

**Issue by kmceres** [gtonkinhill/panaroo#75]
After installing panaroo using conda: `conda install -c conda-forge -c bioconda -c defaults panaroo` when trying to run panaroo I got the following error message: 

```(pangenome) [kristina@stella ~]$ panaroo
Traceback (most recent call last):
  File "/home/kristina/anaconda3/envs/pangenome/bin/panaroo", line 6, in <module>
    from panaroo.__main__ import main
  File "/home/kristina/anaconda3/envs/pangenome/lib/python3.8/site-packages/panaroo/__main__.py", line 16, in <module>
    from .generate_output import *
  File "/home/kristina/anaconda3/envs/pangenome/lib/python3.8/site-packages/panaroo/generate_output.py", line 10, in <module>
    from Bio.Alphabet import generic_dna
  File "/home/kristina/anaconda3/envs/pangenome/lib/python3.8/site-packages/Bio/Alphabet/__init__.py", line 20, in <module>
    raise ImportError(
ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information.
```

It appears that Bio.Alphabet is no longer distributed as part of Biopython.

**Comment by gtonkinhill** [comment]
Hi, thanks for flagging this. We will hopefully push a fix this week.

**Comment by kmceres** [comment]
Thanks! I'll look out for the new release!

**Comment by gtonkinhill** [comment]
This should now be fixed in v1.2.4

**Comment by mabouelk** [comment]
I have the same issue 

ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information.
